lib/migration_helper.rb

```diff
@@ -51,6 +51,25 @@ def estimate_batch_complete(start_time, progress)
   end
   private :estimate_batch_complete
 
+  #
+  # Views
+  #
+
+  def create_view(name, base = "#{name}_base")
```

Comment:
I feel like we should rename this to `create_metrics_view`, only because it's so specific.

Reply:
Or maybe split this into generic and non-generic

```ruby
def create_view(name, query, sequence)
    execute("CREATE VIEW #{name} AS #{query}")
    execute("ALTER VIEW #{name} ALTER COLUMN id SET DEFAULT nextval('#{sequence}')"
end

def create_metrics_view(name)
  base = "#{name}_base"
  create_view(name, "SELECT * FROM #{base}", "#{base}_id_seq")
end
```